### ISSUE TITLE
Diff for upgrading package.json to 0.61.3 suggests old versions

### ISSUE THREAD

**Issue by Brendan-csel** [react-native-community/upgrade-helper#103]
# Bug

React Native 0.61.3 has a dependency on React 16.9.0 but the [upgrade suggestions for 0.61.3](https://react-native-community.github.io/upgrade-helper/?from=0.61.2&to=0.61.3 ) indicate we should use React 16.8.1 (and some other older dependencies)...

![image](https://user-images.githubusercontent.com/1744490/67820661-0c464300-fb1f-11e9-9bcc-b264ddec75e7.png)

## React Native versions

0.61.3

## Steps to reproduce

[Review the suggested diffs for 0.61.3 here](https://react-native-community.github.io/upgrade-helper/?from=0.61.2&to=0.61.3 )

Describe what you expected to happen:

1. expect the dependencies to be at least the same as 0.61.3 and same as `react-native init` produces

PS. sorry if I'm missing something obvious - this is my first attempt at using this tool.  All assistance is greatly appreciated.

**Comment by Brendan-csel** [comment]
I think this is the same issue as [this one](https://github.com/react-native-community/rn-diff-purge/issues/27)

**Comment by kelset** [comment]
hey - yeah we've had that issue in the past, it has something to do with the CLI 😓 we'll try to fix asap

**Comment by kelset** [comment]
cc @pvinis @lucasbento

**Comment by pvinis** [comment]
this is something we are fixing. there later versions are the correct ones, but the template and the cli produced this. I would suggest skipping .3 and waiting for .4 which should have the later versions again.

**Comment by friederbluemle** [comment]
It has been happening for the past several versions.. :/ Thanks everyone for your effort to fix this! 👍

**Comment by pvinis** [comment]
I will close this for now. Let's revisit after .4 if the problem comes back!

**Comment by friederbluemle** [comment]
Any plans to (manually?) fix the diff that is shown for [0.61.2..0.61.3](https://react-native-community.github.io/upgrade-helper/?from=0.61.2&to=0.61.3)

**Comment by pvinis** [comment]
Nope. The problem was .2 and before, and .4 is coming out very soon with a fixed template, so we can just keep going from there.

**Comment by Brendan-csel** [comment]
Thanks for giving this some attention.  Just confirming though that the issues is showing for .3 (currently latest) and seems to be fine for those prior.

**Comment by pvinis** [comment]
Yes. We generated the .0, .1 and .2 using the old cli. For some reason that had the accidental result of using the correct react version. .3 used the new cli, as it should. That surfaced a "bug" in the template, where the versions were not upgraded. This has been addressed and it will be out soon with .4.

Basically, if people are upgrading to 0.61 after .4 is out, all is well. If people have upgraded up to and including .2, all is well (accidentally). If people try to upgrade to .3 and they are already on 0.61, they hopefully will know to not downgrade versions. If they try to upgrade to .3 and they are on <= 0.60, then they will have old versions until .4 is out.

Not the idea situation, but the root cause is being fixed on .4, and the problem with old versions in .3 is not a huge deal for most people.
